lib/rubocop/cop/rails/presence.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks code that can be written more easily using
+      # `Object#presence` defined by ActiveSupport.
+      #
+      # @example
+      #   # bad
+      #   a.present? ? a : nil
+      #
+      #   # bad
+      #   !a.present? ? nil : a
+      #
+      #   # bad
+      #   a.blank? ? nil : a
+      #
+      #   # bad
+      #   !a.blank? ? a : nil
+      #
+      #   # good
+      #   a.presence
+      #
+      # @example
+      #   # bad
+      #   a.present? ? a : b
+      #
+      #   # bad
+      #   !a.present? ? b : a
+      #
+      #   # bad
+      #   a.blank? ? b : a
+      #
+      #   # bad
+      #   !a.blank? ? a : b
+      #
+      #   # good
+      #   a.presence || b
+      class Presence < Cop
+        MSG = 'Use `%s` instead of `%s`.'.freeze
```

Comment:
Small nit. But we're currently deprecating empty string templates for annotated string templates. This common message normally uses 'prefer' and 'current', but its up to you.  This would look like:

```ruby
MSG = 'Use `%<prefer>s` instead of `%<cirrent>s`.'.freeze
```